task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -121,27 +121,32 @@ spec:
         # copy content of conffolder to confdir - will be used in next task - related image check
         cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
 
-        # We have totally 4 checks here currently
-        check_num=4
+        EXTRACT_DIR="/extracted_base_img"
+        mkdir "${EXTRACT_DIR}"
+        oc image extract ${BASE_IMAGE} "${EXTRACT_DIR}"
+        OPM_BINARIES="$(find "${EXTRACT_DIR}" -type f -name opm)"
+        if [[ $(wc -l <<< "$${OPM_BINARIES}") -ne "1" ]]; then
+            echo "expected exactly one opm binary in base image."
```

Comment:
In case of error, you should return error OUTPUT:
```
          ERROR_OUTPUT=$(make_result_json -r ERROR -t "$note")
          echo "${ERROR_OUTPUT}" | tee $(results.TEST_OUTPUT.path)
```
where `$note` is user friendly message about what went wrong